astropy/extern/ply/lex.py

```diff
@@ -41,8 +41,6 @@
 import os
 import inspect
 
-from ...extern.six.moves import zip
```

Comment:
ah, haven't noticed this was an external package when I added this. Could you re-add this?